pibooth/camera/base.py

```diff
@@ -29,10 +29,10 @@ def _hide_overlay(self):
         if self._overlay is not None:
             self._overlay = None
 
-    def _post_process_capture(self, capture_path):
-        """Rework and return a Image object from file.
+    def _post_process_capture(self, capture_data):
```

Comment:
Maybe add a docstring to explain the aim of this method, now that it can raise a not implemented error